File: test_data.rb
require "mysportsfeeds"
require_relative 'StringFind.rb'

PATH = '/Users/donald/Projects/Football-Model'

msf = MySportsFeeds.new(version="1.0", true)
pwd = "2F6F490793BE248DE3B200CF16C545A0"
key = File.read("#{PATH}/key.txt")

msf.authenticate("76991493-077a-4a0f-92da-874b8c", pwd.decrypt(key))

data = msf.msf_get_data('nfl', '2015-2016-regular', 'cumulative_player_stats', 'json', 'team' => 'detroit-lions')

File: test_data.py
from ohmysportsfeedspy import MySportsFeeds
from cryptography.fernet import Fernet
import json

with open('key.txt', 'r') as myfile:
  key = myfile.read()
f = Fernet(key)
token = 'gAAAAABboxejr3hkCDwGeNSEQDPjLbw2T-M41JclD-7ZqqhZWAM61cjqybisLMBFdARpk4L0NsX90tpRNabJcr-oP31W--eiqcbbE8Qe5ZOz9273CdNy5Ts='

msf = MySportsFeeds(version="2.0", store_type='file', store_location='results/')

msf.authenticate(f.decrypt(token), "MYSPORTSFEEDS")
output = msf.msf_get_data(league='nba',season='2016-playoff',feed='seasonal_games',format='json')
print output